tests/test_installation.py

```diff
@@ -0,0 +1,27 @@
+from pytest_bdd import scenario, given, when, then
+from pytest_bdd import parsers
+import os
+import subprocess
+
+@scenario('installation.feature', 'Run basic installation')
+def test_installation():
+    pass
+
+@given('A complete inventory')
+def inventory_bdd(inventory):
+    return inventory
+
+@when("I run 'make shell'")
+def run_make_shell():
+    subprocess.check_call('make shell < /dev/null', shell=True)
```

Comment:
So, we're not really using the `make shell` env. I think this is wrong. Is it possible to launch `make shell` in a subprocess, then keep a reference to it and have e.g. the `I launch the ... playbook` use this subprocess and inject a call to `ansible-playbook ...`?